caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -359,7 +359,7 @@ def _handle_predict_exceptions(self, model_id: str, request_name: str):
                 model_id=model_id,
             ).inc()
             raise CaikitRuntimeException(
-                StatusCode.INTERNAL, "Unhandled exception during prediction"
+                StatusCode.INTERNAL, f"{e}",
```

Comment:
don't really need the f-string?

Reply:
It seems we do, as removing it causes "Internal server error" instead of the exception